Makefile

```diff
@@ -23,7 +23,11 @@ FILES     := $$(find . -name '*.go' -type f | grep -vE 'vendor')
 VENDOR_TIDB := vendor/github.com/pingcap/tidb
 
 
-build: check test importer checker dump_region binlogctl sync_diff_inspector
+build: prepare check test importer checker dump_region binlogctl sync_diff_inspector finish
+
+prepare:		
+	mv go.mod1 go.mod
+	mv go.sum1 go.sum
```

Comment:
If any step between `prepare` and `finish` failed these will not be moved back.

Perhaps:

1. use `cp` instead of `mv`
2. delete the `finish` step
3. put `go.mod` and `go.sum` into `.gitignore`

Reply:
I think the first one is better

Reply:
I mean do all three 😅

Reply:
but if delete the finish step, we need rename it by manual，seems not suitable.

Reply:
done 1 and 3

Reply:
2 copy xx to xx in finish

Reply:
@kennytm 
The root cause of this problem is semantic version.
If the code can't guarantee semantic version, `go.mod` can't help.